sekoia_automation/scripts/documentation/generate.py

```diff
@@ -132,6 +136,41 @@ def _copy_logo(self, module_path: Path, module_manifest: dict) -> str | None:
                 return module_logo_filename
         return None
 
+    def _copy_configure_md(
+        self, module_path: Path, module_manifest: dict
+    ) -> str | None:
+        """
+        Copy and return the CONFIGURE.md content
+
+        Read the content of CONFIGURE.md in the automation module directory.
+        Copy the items in docs/assets into the documentation and replace the
+        paths in the content.
+        """
+        # Is the SETUP.md exists
+        if not (module_path / "CONFIGURE.md").exists():
+            return None
+
+        # Read it content
+        with (module_path / "CONFIGURE.md").open("r") as fd:
+            content = fd.read()
+
+        # Copy the assets for the documentation and update the path
```

Comment:
Maybe the assets  copy should be independant from the configure.md in case we want to also include images in the description.